tests/integration/components/common/busy-model-wrapper-test.js

```diff
@@ -1,53 +1,46 @@
 import { moduleForComponent, test } from 'ember-qunit';
 import hbs from 'htmlbars-inline-precompile';
-import pageObject from 'ember-cli-page-object';
-import busyModelWrapper from 'code-corps-ember/tests/integration/components/common/busy-model-wrapper-test';
 import { set } from '@ember/object';
 
-let page = pageObject.create(busyModelWrapper);
-
-function renderPage() {
-  page.render(hbs`
-  {{common/busy-model-wrapper
-    model=model
-  }}`);
-}
-
 moduleForComponent('common/busy-model-wrapper', 'Integration | Component | common/busy model wrapper', {
-  integration: true,
-  beforeEach() {
-    page.setContext(this);
-  },
-  afterEach() {
-    page.removeContext();
-  }
+  integration: true
 });
 
-test('it shows "saving" if model is saving, but not deleted', function(assert) {
-  let model = { isSaving: true, isDeleted: false };
-  set(this, 'model', model);
-  renderPage();
+// These tests do not make use of a page object. One was written, but it consisted of an empty string and a .busymodelwrapper scope. Set up this way, the page object wasn't actually rendering anything, just empty code placeholders. Switched to using jQuery selectors in order to have cleaner more effectived code.
```

Comment:
Biggest thing I want attention on is this comment. Want to make sure it makes sense. Remember, we veto'd the page object for this test because it was just, if I remember right:
`tagname: ' ' 
 scope: .busymodelwrapper`